clients/cobol-lsp-vscode-extension/src/__tests__/services/copybook/CopybookResolveURITest.spec.ts

```diff
@@ -312,4 +312,21 @@ describe("Prioritize search criteria for copybooks test suite", () => {
     expect(spySearchInWorkspace).toBeCalledTimes(1);
     (globSync as any) = jest.fn().mockReturnValue((x: any) => x);
   });
+  test("Local folder with a relative path given at settings.json, performs search local respective folder", async () => {
+    vscode.workspace.getConfiguration = jest.fn().mockReturnValue({
+      get: jest.fn().mockReturnValue([CPY_FOLDER_NAME]),
+    });
+
+    (globSync as any) = jest.fn().mockReturnValue([copybookName]);
```

Comment:
This mock is too simplistic for the purpose... It should find the copybookName only when "/storagePath" and "/.cobcopy" are both present in the path.

And the expectation is that the `resolveCopybookHandler` should not find it

Reply:
"/storagePath" is always provided and which is a imitator for global storage uri that is always available.
 This test checks if relative path provided at configuration settings and search is performed at that location wrt current open workspace ( as if ).

Reply:
Yes, but we are missing the test for the opposite behavior, the function should not be using the relative paths to search in the download folder.

Reply:
not really. `searchCopybookInExtensionFolder` isn't performed here.

check this line : https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/pull/2265/files#diff-766816dda15ebed740ac2ef0bbcb8c0a6230371bee854c30a5e1406f49755a84R36

Reply:
That's the problem, I am interested in testing the following chain: `resolveCopybookHandler -> searchCopybookInExtensionFolder -> globSearch -> isAbsolute(resource) == false -> globSync`

Reply:
not sure this resolves :  3d0468d76b5639ed8fb6a73b60eeac0b21a31b1b

do you think we should also check isAbsolute is returned false ?